public/client/TracyCallstack.cpp

```diff
@@ -93,6 +93,14 @@ extern "C" const char* ___tracy_demangle( const char* mangled )
 namespace tracy
 {
 
+// when "TRACY_SYMBOL_OFFLINE_RESOLVE" is set to "1", instead of fully resolving symbols at runtime,
+// simply resolve the offset and image name (which will be enough the resolving to be done offline)
+bool getDoOfflineSymbolResolve()
```

Comment:
It would probably be a good idea to add a `#ifdef TRACY_SYMBOL_OFFLINE_RESOLVE` to enable hardcode enabling this at build time, like you can do with some other options. Also, rename this function to `ShouldResolveSymbolsOffline`.

Reply:
makes sense, I'll try to add support for the compile time ifdef in another commit later today/tomorrow as it will require a bit of #ifdefing in that file to make sure the libbacktrace/dbghelper is ifdefed out

Reply:
Add the ifdef as an option to the user, but also keep the current env variable based solution.

Reply:
Ah, I see you already committed the changes. I'm not sure it's worth having ifdefs scattered all over the code. Sure, it's dead code, but it also increases the mental effort required to understand what's going on.